pkg/base1/cockpit.js

```diff
@@ -2366,6 +2366,8 @@ function full_scope(cockpit, $, po) {
             return _("The server refused to authenticate using any supported methods.");
         else if (problem == "unknown-hostkey")
             return _("Untrusted host");
+        else if (problem == "changed-hostkey")
+            return _("Host key changed");
```

Comment:
Shouldn't this be 'invalid-hostkey' and "Host key is incorrect" or "Host key is invalid". "Changed" doesn't sound like a problem to the user.